concordium-sdk/src/main/java/com/concordium/sdk/crypto/encryptedtransfers/GlobalContext.java

```diff
@@ -0,0 +1,33 @@
+package com.concordium.sdk.crypto.encryptedtransfers;
+
+import com.concordium.sdk.crypto.bulletproof.BulletproofGenerators;
+import com.concordium.sdk.crypto.pedersencommitment.PedersenCommitmentKey;
+import com.concordium.sdk.responses.cryptographicparameters.CryptographicParameters;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+class GlobalContext {
```

Comment:
Documentation. When is this used, what is the lifetime? e.g..

Reply:
done